modules/retail/pages/retail-install-setup.adoc

```diff
@@ -47,52 +74,113 @@ For more information about formulas, see xref:retail:retail-formulas-intro.adoc[
+For dedicated network topology run this command:
++
+----
+retail_branch_init <branch_server_minion_id> --dedicated-nic <network_device>
+----
++
+You can customize network information as well, together with custom [systemitem]``branch prefix``. For example:
++
+----
+retail_branch_init <branch_server_minion_id> --dedicated-nic eth1
+                                             --branch-prefix B001
+                                             --server-domain <branch_server_subdomain>
+                                             --branch-ip 192.168.86.1
+                                             --netmask 255.255.255.0
 ----
 +
-This command will configure branch server formulas with recommended values.
 You can use the [command]``retail_branch_init --help`` command for additional options.
+
 . Verify that your changes have been configured correctly by checking the {susemgr} {webui} branch server system formulas.
 . Apply highstate on the branch server.
 You can do this through the {webui}, or by running this command:
 +
 ----
-salt <branch_server_salt_id> state.apply
+salt <branch_server_minion_id> state.apply
 ----
 
+Similar results can be achieved by using mass import command line tool.
 
-== Synchronize Images to the Branch Server
+.Procedure: Configuring Branch Server Formulas With a Mass Import Tool
 
-The OS image you use on the {susemgr} server must be synchronized for use on the branch server.
-You can do this with the Salt [command]``image-sync`` tool.
-
-.Procedure: Synchronizing Images to the Branch Server
-
-. On the {susemgr} server, run this command:
+. Prepare branch specific YAML file:
++
+For example, create branch.yaml file with content:
 +
 ----
-salt <branch_server_salt_id> state.apply image-sync
+branches:
+  <branch_server_minion_id>:
+    branch_prefix: branch1
+    server_name: branchserver1
+    server_domain: example.com
+    nic: eth1
+    dedicated_nic: true
+    configure_firewall: true
+    branch_ip: 192.168.2.1
+    netmask: 255.255.255.0
+    dyn_range:
+        - 192.168.2.10
+        - 192.168.2.250
 ----
-. The image details will be transferred to [path]``/srv/saltboot`` on the branch server.
++
+For more information about mass import tool, see xref:retail:retail-mass-config.adoc[].
+. Import branch information from YAML file to {susemgr}
++
+----
+retail_yaml --from-yaml branch.yaml
+----
+. Verify that your changes have been configured correctly by checking the {susemgr} {webui} branch server system formulas.
+. Apply highstate on the branch server.
 
+[WARNING]
+====
+Both [command]``retail_branch_init`` and [command]``retail_yaml`` commands overwrites existing configuration of specified branch server.
+====
 
+After the initial configuration done by command line tools, branch server configuration can be further adjusted in {susemgr} {webui} through branch server formulas.
 
-== Terminals Based on SLES{nbsp}11 SP{nbsp}3
+=== Create Required System Groups
```

Comment:
This type of heading pre-supposes a procedure, but there isn't one ...

Reply:
Right, I'll add a procedure header below.

Reply:
At the end I changed the header to `Required System Groups`. I couldn't figure out what procedure to add as I am xrefing to the other part of doc for howto.